cli/kata-check_arm64.go

```diff
@@ -5,6 +5,37 @@
 
 package main
 
+import "github.com/sirupsen/logrus"
+
+const (
+	cpuFlagsTag        = "Features"
+	archCPUVendorField = "CPU implementer"
```

Comment:
About the implementer here, I think Arm and x86 may have some differences here. The CPU implementer here for Arm means the CPU core IP implementer, not the SoC vendor. For example, if Apple and Qualcomm use the Arm's IP implementation to design their SoCs (In actually, they use their own implementations :) ), the CPU implementer here would be the same for these two SoC vendors. So, if the archCPUVendorField presents the CPU core vendor, I think you can use CPU implementer here. How does x86 use the archCPUVendorField?

Reply:
On `x86_64`, this information is used to display some useful info in the output of `kata-env`:

```
  [Host.CPU]
    Vendor = "GenuineIntel"
    Model = "Intel(R) Core(TM) i7-5600U CPU @ 2.60GHz"
```

I wasn't really sure how to make that relevant for ARM, so any advice would be much appreciated ;)